### DISCUSSION TITLE
OKD IPI installer does not recognise private hosted zone on AWS

### DISCUSSION THREAD

**Discussion by annemsubba** [okd-project/okd#1219] [Q&A]
**Describe the bug** 
--------------------------
while run  the openshift-installer create cluster --log-level=info  i can only see the public hosted zone and i unable to see the private hosted zone in AWS.

**Version**
-----------------
OKD version release: 4.10.0-0.okd-2022-05-07-021833
IPI install method and the cloud platform  AWS used 

**How reproducible**
-----------------------------
 i'm creating the install-config.yaml there i mentioned the private and aws configurations
1) i created the VPC with CIDR 10.0.0.0/16
2)i created the public subnet and private subnet different available zones 
3) i created the route table for  subnet and private subnet 
4)i created the IGW and also NAT gateway
5) i created the private hosted zone in the route53
6)i created the endpoint services like ec2, Load balancer, VPC service endpoint 
7)i have mentioned the private subnets in the install-config file 
8)configured the VPC CIDE in the install-config file and also mentioned the endpoint
9)in the given below the install-config.ymal file with the private cluster

apiVersion: v1
baseDomain: example.com
compute:
- architecture: amd64
  hyperthreading: Enabled
  name: worker
  platform: {}
  replicas: 3
controlPlane:
  architecture: amd64
  hyperthreading: Enabled
  name: master
  platform: {}
  replicas: 3
metadata:
  creationTimestamp: null
  name: dev
networking:
  clusterNetwork:
  - cidr: 192.252.0.0/14
    hostPrefix: 23
  machineNetwork:
  - cidr: 192.168.0.0/16
  networkType: OVNKubernetes
  serviceNetwork:
  - 172.30.0.0/16
platform:
  aws:
    region: ap-south-1
    userTags:
      adminContact: jdoe
      costCenter: 7536
    subnets: 
    - private-subnet1
    - private-subnet2
    - privte- subnet
    -     amiID: ami-00016509ad9123435
    serviceEndpoints: 
      #- name: ec2
      #  url: https://vpce-id.ec2.ap-south-1.vpce.amazonaws.com
      #- name: lb
      #  url: https://vpce-id.elasticloadbalancing.ap-south-1.vpce.amazonaws.com
      - name: service
        url: https://vpce-id-svc-0186bbaaf156cd9e4.ap-south-1.vpce.amazonaws.com     
    hostedZone: Z0SDF3DF3973
fips: false 
publish: Internal
pullSecret: ........
sshKey: |.........
  

After run the openshift-installer create cluster
 i'm not not able to proceed the next step its still shown the loading the platform and also the end point are not connected its getting connection timeout error.

**Log bundle**
----------------------------
time="2022-05-10T18:32:22+05:30" level=debug msg="OpenShift Installer 4.10.0-0.okd-2022-05-07-021833"
time="2022-05-10T18:32:22+05:30" level=debug msg="Built from commit 0adfce2a1bfede7d74ac8930a5d56df1a406a533"
time="2022-05-10T18:32:22+05:30" level=debug msg="Fetching Metadata..."
time="2022-05-10T18:32:22+05:30" level=debug msg="Loading Metadata..."
time="2022-05-10T18:32:22+05:30" level=debug msg="  Loading Cluster ID..."
time="2022-05-10T18:32:22+05:30" level=debug msg="    Loading Install Config..."
time="2022-05-10T18:32:22+05:30" level=debug msg="      Loading SSH Key..."
time="2022-05-10T18:32:22+05:30" level=debug msg="      Loading Base Domain..."
time="2022-05-10T18:32:22+05:30" level=debug msg="        Loading Platform..."
time="2022-05-10T18:32:22+05:30" level=debug msg="      Loading Cluster Name..."
time="2022-05-10T18:32:22+05:30" level=debug msg="        Loading Base Domain..."
time="2022-05-10T18:32:22+05:30" level=debug msg="        Loading Platform..."
time="2022-05-10T18:32:22+05:30" level=debug msg="      Loading Networking..."
time="2022-05-10T18:32:22+05:30" level=debug msg="        Loading Platform..."
time="2022-05-10T18:32:22+05:30" level=debug msg="      Loading Pull Secret..."
time="2022-05-10T18:32:22+05:30" level=debug msg="      Loading Platform..."

**Comment by vrutkovs** [comment]
Please attach [log bundle](https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting)

**Comment by annemsubba** [comment]
Hi,
please find the below atteched document


On Wed, May 11, 2022 at 12:08 PM Vadim Rutkovsky ***@***.***>
wrote:

> Please attach log bundle
> <https://docs.okd.io/latest/installing/installing-troubleshooting.html#installation-bootstrap-gather_installing-troubleshooting>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/okd/discussions/1219#discussioncomment-2727695>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUHZ3GGDVTAYLKUFB4SPFO3VJNIXBANCNFSM5VTZZECQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Thanks & Regard
Annem Subba Reddy

*Walking Tree Technologies Pvt Ltd*
Phone: 9121438982
***@***.***
